### ISSUE TITLE
CodeSnippet v10: differences with the carbon design

### ISSUE THREAD

**Issue by ZrianinaMariia** [carbon-design-system/carbon-components-vue#314]
## Detailed description

1) Carbon react has knobs for 'Copied text', vue version has fixed text.
2) Carbon react provides light theme for inline code snippet.
3) 'Copy' icon  should have hover state (blue border) in addition to active state.
4) Inline long code snippets: did not see examples from carbon design or carbon react, but the tooltip with 'copied' looks a bit off.

5) Multiline code snippet: 'showLess' and 'snowMore' buttons are not correctly aligned, i.e. should be on the right and not left.
6) Multiline vs multiline minimal do not differ. Expect multiline minimal be displayed without 'showMore/showLess' buttons.
7) White background in stories for multiline and multiline minimal is  missing.

![image](https://user-images.githubusercontent.com/5481483/55565844-e4af0980-56fa-11e9-9a20-c030ad16b5ab.png)

#306

**Comment by ZrianinaMariia** [comment]
Thanks, Lee! 

5. I did not refer specifically to CvButton, but more to the layout of the 'Show Less' and 'show more' text, i.e. 
carbon-vue:
![image](https://user-images.githubusercontent.com/5481483/55953601-d01fc380-5c5c-11e9-827a-4b82bc012122.png)

carbon-design:
![image](https://user-images.githubusercontent.com/5481483/55953646-f5accd00-5c5c-11e9-9922-df2c539553fc.png)

I just checked the latest carbon-vue and it looks like the problem has been fixed.

6. Sorry, I did not understand your answer.

7. What I meant is that it is hard to see the gray component on the gray story background. It would be great if the story background color differs from the component background color, so that it is easier to see where the component starts and where it ends.
